pkg/cloudprovider/vsphere/loadbalancer/config/config_ini_legacy_test.go

```diff
@@ -62,36 +66,36 @@ host = nsxt-server
 	assertEquals("LoadBalancer.tcpAppProfileName", config.LoadBalancer.TCPAppProfileName, "default-tcp-lb-app-profile")
 	assertEquals("LoadBalancer.udpAppProfileName", config.LoadBalancer.UDPAppProfileName, "default-udp-lb-app-profile")
 	assertEquals("LoadBalancer.size", config.LoadBalancer.Size, "MEDIUM")
-	if len(config.LoadBalancerClasses) != 2 {
-		t.Errorf("expected two LoadBalancerClass subsections, but got %d", len(config.LoadBalancerClasses))
+	if len(config.LoadBalancerClass) != 2 {
+		t.Errorf("expected two LoadBalancerClass subsections, but got %d", len(config.LoadBalancerClass))
 	}
-	assertEquals("LoadBalancerClass.public.ipPoolName", config.LoadBalancerClasses["public"].IPPoolName, "poolPublic")
-	assertEquals("LoadBalancerClass.private.tcpAppProfileName", config.LoadBalancerClasses["private"].TCPAppProfileName, "tcp2")
-	assertEquals("LoadBalancerClass.private.udpAppProfileName", config.LoadBalancerClasses["private"].UDPAppProfileName, "udp2")
+	assertEquals("LoadBalancerClass.public.ipPoolName", config.LoadBalancerClass["public"].IPPoolName, "poolPublic")
+	assertEquals("LoadBalancerClass.private.tcpAppProfileName", config.LoadBalancerClass["private"].TCPAppProfileName, "tcp2")
+	assertEquals("LoadBalancerClass.private.udpAppProfileName", config.LoadBalancerClass["private"].UDPAppProfileName, "udp2")
 	if len(config.LoadBalancer.AdditionalTags) != 2 || config.LoadBalancer.AdditionalTags["tag1"] != "value1" || config.LoadBalancer.AdditionalTags["tag2"] != "value 2" {
 		t.Errorf("unexpected additionalTags %v", config.LoadBalancer.AdditionalTags)
 	}
-	assertEquals("NSX-T.user", config.NSXT.User, "admin")
-	assertEquals("NSX-T.password", config.NSXT.Password, "secret")
-	assertEquals("NSX-T.host", config.NSXT.Host, "nsxt-server")
+	assertEquals("NSXT.user", config.NSXT.User, "admin")
+	assertEquals("NSXT.password", config.NSXT.Password, "secret")
+	assertEquals("NSXT.host", config.NSXT.Host, "nsxt-server")
 }
 
-func TestReadConfigOnVMC(t *testing.T) {
+func TestReadINIConfigOnVMC(t *testing.T) {
 	contents := `
 [LoadBalancer]
-ipPoolID = 123-456
+ip-pool-id = 123-456
```

Comment:
For posterity: this is not a breaking change since we haven't release a version supporting this yet. 

cc @mandelsoft @MartinWeindel

Reply:
FYI, the README (which I missed initially but caught in this pass) have been updated to default to YAML and the `go test` for the LB config was originally for INI and is now YAML (passes).